src/webgpu/web_platform/copyToTexture/ImageBitmap.spec.ts

```diff
@@ -119,6 +119,30 @@ g.test('from_ImageData')
   Test ImageBitmap generated from ImageData can be copied to WebGPU
   texture correctly. These imageBitmaps are highly possible living
   in CPU back resource.
+
+  It generates pixels in ImageData one by one based on a color list:
+  [RED, GREEN, BLUE, WHITE, OPAQUE_BLACK, TRANSPARENT_BLACK].
```

Comment:
It seems like the names of these colors are wrong. White is [0, 0, 0, 1], OpaqueBlack is [1, 1, 1, 1], and TransparentBlack is [1, 1, 1, 0.6].

I've renamed them and updated these two descriptions.
Black, White, SemitransparentWhite.